src/components/Cards/PreviewCard/PreviewCard.module.scss

```diff
@@ -80,7 +81,7 @@
 
 	.Image > img {
 		width: 100%;
-		height: 100%;
```

Comment:
Removing `height: 100%` is breaking the image.

![image](https://user-images.githubusercontent.com/12437916/141203469-f317a9fe-eb85-4a9e-bd18-f277f6c422b3.png)

I'll have to circle back on how we're going to get this padding consistent. I imagine we'll have to do a similar check as we're doing in NFT Card, and hardcode aspect ratios based on domain name (for now)

Reply:
Hey Brett, Thank you. I asked Ethan about this image. I think he replied about this on the card itself. But yeah, I thought this one would be a tricky one to get right.

Reply:
I've put the height back on. What's altering this it's the max height.

Reply:
I've checked with Ethan. And he said that it was ok. So yeah, circle back on with him to see what the right approach is regarding this.